javascript/javascript2/week2/hyfBay week-2/main.js

```diff
@@ -0,0 +1,47 @@
+console.log("Script loaded");
+const contentUl = document.getElementById("contents");
+const search = document.getElementById("search");
+
+let products = getAvailableProducts();
+console.log(products);
+
+search.addEventListener('input', e =>{
+    console.log("searhed: " + search.value);
+    let filteredProducts = products.filter(p =>{
+        let item = p.name.toLowerCase();
+        if(item.includes(search.value.toLowerCase())){
+            console.log(p);
+            return true;
+        }else{
+            return false;
+        }
```

Comment:
Good solution, here's a shorter one:
```
products.filter(p =>{ return p.name.toLowerCase().includes(search.value.toLowerCase())
```
Also, the filteredProducts could be a const